src/app/rundown/components/rundown/rundown.component.html

```diff
@@ -1,30 +1,24 @@
-<body>
-<div class="title">{{rundown?.name}}</div>
+<sofie-rundown-header *ngIf="rundown" class="c-rundown-header" [rundown]="rundown"></sofie-rundown-header>
+
+<div *ngIf="rundown?.isActive" class="test-buttons">
+    <ng-container *ngIf="!rundown?.isActive; else activeRundownButtons"></ng-container>
+</div>
```

Comment:
If you negate the if-statement on line 3 then you can complete remove the if-statement on line 4 and write:
```
<div *ngIf=!rundown?.isActive ...>
  <button ...>Take Next</button>
</div>
```

Reply:
I did exactly that, great suggestion!